scripts/update_baseline_dependency_layers.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+# Part of genivi-dev-platform
+# (C) Gunnar Andersson
+# License: MPLv2
+
+# Update poky, etc. to the versions required by the current meta-ivi baseline
+
+fail() {
+  echo "Something failed.  You might need to clean the layer somehow (see above).  Giving up."
+  exit 1
+}
+
+update_version() {
+  layer=$1
+  cd meta-ivi
+  v=$(scripts/get_layer_info_from_README.sh $layer revision)
```

Comment:
I can't find the get_layer_info_from_README.sh script anywhere, is it made available through a different PR?

Reply:
Ah,  thanks! That makes sense I guess ;)

Reply:
Yes, especially since the line before it is "cd meta-ivi" ;)

Reply:
Rub it in my face, why dont you ;)